### PR TITLE
Adding NDI5Runtime 5.5.2.0

### PR DESCRIPTION
**Pull request by Trouffman**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

Adding NDI 5 Runtime using the official Download link from Newtek/NDI.

This is filed under "Newtek" to follow with pre-existing manifest from NDI Tools.

Licence "MIT" selected as this is supposedly the same as the NDI Tools.

Test tried :
Tested with the regular install & with the winget --silent argument as well.
This was tested on a semi-clean machine (Do not have the sandbox available on the machine)

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/94292)

### CODE REVIEW

**Comment by Trouffman** [comment]
Rookie mistake initially : i manually moved the file after the test....
image

This is the test result on the build machine (not in sandbox)

File: manifests/n/NewTek/NDI5Runtime/5.5.2.0/NewTek.NDI5Runtime.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+PackageVersion: 5.5.2.0
+MinimumOSVersion: 10.0.0.0
+InstallerType: inno
+Scope: machine
+InstallModes:
+- interactive
+- silent
+InstallerSwitches:
+  Silent: /VERYSILENT
+  SilentWithProgress: /SILENT
```
Comment (mdanish-kh) [inline manifests/n/NewTek/NDI5Runtime/5.5.2.0/NewTek.NDI5Runtime.installer.yaml:14]:
```suggestion
```
Not required when InstallerType is set to Inno. WinGet will automatically pass these switches to the installer.

File: manifests/n/NewTek/NDI5Runtime/5.5.2.0/NewTek.NDI5Runtime.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+UpgradeBehavior: install
+Installers:
+- Architecture: x64
+  InstallerUrl: https://downloads.ndi.tv/SDK/NDI_SDK/NDI%205%20Runtime.exe
+  InstallerSha256: 8D2E8D32C62F4C34809EF77DF432B85DE35978B12358DF19889D723D0579F2BC
+  ProductCode: '{CF8E6E00-9C03-4440-81C0-21FACB921A6B}'
```
Comment (mdanish-kh) [inline manifests/n/NewTek/NDI5Runtime/5.5.2.0/NewTek.NDI5Runtime.installer.yaml:20]:
```suggestion
  ProductCode: '{71AFF296-ED43-4166-8301-4649285EE712}_is1'
```
Installer writes uninstall entries to this location in the registry

File: manifests/n/NewTek/NDI5Runtime/5.5.2.0/NewTek.NDI5Runtime.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,31 @@
+# Created with YamlCreate.ps1 v2.2.1 $debug=NVS1.CRLF.5-1-22621-963.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: NewTek.NDI5Runtime
+PackageVersion: 5.5.2.0
+PackageLocale: en-US
+Publisher: Newtek, Inc.
```
Comment (mdanish-kh) [inline manifests/n/NewTek/NDI5Runtime/5.5.2.0/NewTek.NDI5Runtime.locale.en-US.yaml:7]:
```suggestion
Publisher: NewTek, inc.
```
Must match Control Panel  / Apps&Features

File: manifests/n/NewTek/NDI5Runtime/5.5.2.0/NewTek.NDI5Runtime.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+PublisherUrl: https://ndi.tv/
+# PublisherSupportUrl:
+# PrivacyUrl:
+# Author:
+PackageName: NDI 5 Runtime
+PackageUrl: https://ndi.tv/
+License: MIT
```
Comment (mdanish-kh) [inline manifests/n/NewTek/NDI5Runtime/5.5.2.0/NewTek.NDI5Runtime.locale.en-US.yaml:14]: Can you provide a link to the license? I can't seem to find it on the PackageUrl https://ndi.tv/. If no info about the Licensing can be found, this field should be set to `Proprietary`.

**Review by mdanish-kh — CHANGES_REQUESTED** [review]

File: manifests/n/NewTek/NDI5Runtime/5.5.2.0/NewTek.NDI5Runtime.installer.yaml (hunk 2 above)
Comment (Trouffman) [reply manifests/n/NewTek/NDI5Runtime/5.5.2.0/NewTek.NDI5Runtime.installer.yaml]: Indeed, I did cross-check on my machine. 
Full path if this is needed later :
HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\{71AFF296-ED43-4166-8301-4649285EE712}_is1

**Review by Trouffman — COMMENTED** [review]
Thanks for the clear comments, easier to see what needs to be changed / tweaked for future packages submissions.

**Review by mdanish-kh — APPROVED** [review]
Note: The pipelines are down at the moment (https://github.com/microsoft/winget-pkgs/pull/94292#issuecomment-1398425480), so this package would not be available through `winget install` until this PR passes the publish pipeline post-merge.
The PR will get labeled with `Publish-Pipeline-Succeeded` when the pipeline is up again and the package passes successfully though it.

**Pull request merged**